convergence.md

```diff
@@ -1,3 +1,501 @@
+
+### Merge Process
+
+Of the two existing repositories, `iojs/io.js` has been the most active since the fork. Since that time, there has been some cherry-picking of commits between the two projects but the activity has been inconsistent and the separate code bases have diverged in fairly significant ways. Based on the commit history, the easiest merge approach will be to create the new `nodejs-foundation/node` repository as a Fork of the existing `iojs/io.js` (`nodejs-foundation/io.js`) repository, then cherry pick commits from `joyent/node` (`nodejs-foundation/node`) to merge in. Unfortunately, this will not be as easy as simply using `git format-patch` and `git am`.
+
+The commits generally fall into three categories:
+
+* Features
+* Fixes
+* Doc & Test
+
+Feature commits will most likely need to be reimplemented using the original commit as a guide. There are some that may be able to transfer directly but they are likely rare.
+
+Fix commits are tricky because, in many cases, the issue may have already been fixed in io.js but in a different way than what was landed in joyent/node. For these, the process is:
+
+1. determine if there is an existing test case for the fix and if there is not, create one,
+2. test to see if the issue has already been fixed in the io.js base,
+3. if it has, compare the two fixes and determine if there were any additional pieces added to the `joyent/node` fix that need to be added
+4. if it has not been fixed, port the fix from `joyent/node`
+
+Doc & Test commits are handled in generally the same way as Fix commits.
+
+#### Merging from Parallel io.js and node.js tracks
+
+While the repositories are being merged, the existing io.js and node release streams will continue to operate. This implies that there will be new commits to both that will also need to be merged into the converged repo. Those are to be handled in generally the same way as the other commits.
+
+Once the Foundation launches, the decision making process for both release streams follows the Foundation TSC guidelines with the combined TSC overseeing both individual release lines. That said, the TSC can delegate the day-to-day details of managing each release line to the existing individual project TCs.
+
+While work is being done to converge the two lines, care must be taken when landing features or fixes in either line that would impact or conflict with changes being made in the other line. The goal is to avoid a situation where the release lines diverge further *after* the Foundation launch while effort is being made to bring the lines together. When significant changes come in that can have an impact on the larger effort, those should be reviewed by the combined TSC.
+
```

Comment:
I wonder if it's possible that the two projects will continue diverging faster than they will be converging, and what can be done to avoid that. In the worst case scenario, would it make sense to forcibly slow down or freeze work in the individual project lines? And what can we do to track progress?